### ISSUE TITLE
Unable to replicate the results

### ISSUE THREAD

**Issue by sangramkapre** [brucechou1983/CheXNet-Keras#12]
Hi, thanks a lot for providing the Keras implementation of CheXNet.
I trained your model (without any change) on the NIH-14 dataset. But average of class specific AUC values is around 0.68 only. The maximum AUC value achieved for this implementation is about 0.77 (for MASS pathology). These numbers seem significantly lower than those published in the paper, or [PyTorch implementation on Github](https://github.com/arnoweng/CheXNet). Could you please let me know if I am missing anything important here? Thanks!

**Comment by brucechou1983** [comment]
@sangramkapre 

If you clone the project and run it using the default setting, you should be able to see MASS AUC ~ 0.82-0.85 in test set. Maybe try running it for a longer time?
